### DISCUSSION TITLE
Remove Microsoft Namespace

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#9] [General]
How about removing Microsoft namespace and leave just Tye?

**Comment by razvangoga** [comment]
Maybe **_tye2_** - **_tye_** is and will remain the original from MS.

Since Tye was published under a MIT licence (https://github.com/dotnet/tye?tab=MIT-1-ov-file#readme), i think we are allowed to do this rename as long as we remain on the MIT licence.

I've moved the repo under an own organization so, we can have more repos associated with it (i've went you an invite as a contributor).
Sadly **_tye2_** as org name was already taken so i settled on **_tye-v2_** (the org can be renamed if you have any other better ideas :) )

I'm thinking maybe it would be wiser to have the org name in the namespace (just like MS did with Microsoft.Tye)

**Comment by drakon660** [comment]
just remove Microsoft leave Tye as root namespace, tye2.domain is better to remember than tye-v2.

**Comment by drakon660** [comment]
closing
